importer/src/bin/geojson_to_osmosis.rs

```diff
@@ -0,0 +1,61 @@
+use std::io::{self, Read};
+use geojson::{GeoJson, Value};
+
+/// Convert geojson boundary suitable for osmfilter and other osmosis based tools.
+/// Expects the input to no elements other than the boundary of interest.
+//
+/// Reads geojson text from stdin
+/// Writes "poly" formatted text to stdout
+fn main() -> Result<(), Error> {
+    let mut buffer = String::new();
+    io::stdin().read_to_string(&mut buffer)?;
+    let geojson = buffer.parse::<GeoJson>()?;
+    let points = boundary_coords(&geojson)?;
+
+    println!("map_name");
```

Comment:
Does this need to be configurable?

Reply:
Actually no. I've been filling it out manually, but nothing pays attention to this. So this is fine.